lightning/src/ln/mod.rs

```diff
@@ -86,6 +86,15 @@ pub struct PaymentPreimage(pub [u8;32]);
 /// (C-not exported) as we just use [u8; 32] directly
 #[derive(Hash, Copy, Clone, PartialEq, Eq, Debug)]
 pub struct PaymentSecret(pub [u8;32]);
+/// Additional BOLT 11 data to attach to the payment, useful for applications where the recipient
+/// doesn't keep any context for the payment.
```

Comment:
Since the other types here feature the same comment, for the sake of consistency this should probably also include the comment `/// (C-not exported) as we just use [u8; 32] directly`.

Reply:
Ah, good to know there are magic comments! Nevermind then!